apps/SageAccountWeb/src/components/ProfileValidation/ProfileFieldsEditor.tsx

```diff
@@ -83,14 +92,9 @@ export const ProfileFieldsEditor = (props: ProfileFieldsEditorProps) => {
           margin="normal"
           required
         >
-          <TextField
-            fullWidth
-            label={'Current Affiliation'}
-            id="company"
-            name="company"
-            onChange={handleChange}
+          <RORInstitutionField
+            onChange={value => handleCompanyChange(value || '')}
             value={values.company || ''}
-            error={!!errors.company}
```

Comment:
Do we need the RORInstitutionField to handle an error?

Reply:
I think in this specific case, it doesn't actually matter.
1. The validation runs after you submit, and makes sure that the `company` field is not empty
2. We do not let you submit unless the `company` field is not empty.

However I'll add it back just in case this validation changes.